client/src/components/SidePanel/Agents/ModelPanel.tsx

```diff
@@ -1,16 +1,16 @@
 import React, { useMemo, useEffect } from 'react';
 import { ChevronLeft, RotateCcw } from 'lucide-react';
 import { useFormContext, useWatch, Controller } from 'react-hook-form';
-import { getSettingsKeys, alternateName } from 'librechat-data-provider';
+import { getSettingsKeys, alternateName, agentParamSettings, SettingDefinition } from 'librechat-data-provider';
 import type * as t from 'librechat-data-provider';
 import type { AgentForm, AgentModelPanelProps, StringOption } from '~/common';
 import { componentMapping } from '~/components/SidePanel/Parameters/components';
-import { agentSettings } from '~/components/SidePanel/Parameters/settings';
 import ControlCombobox from '~/components/ui/ControlCombobox';
 import { useGetEndpointsQuery } from '~/data-provider';
 import { getEndpointField, cn } from '~/utils';
 import { useLocalize } from '~/hooks';
 import { Panel } from '~/common';
+import _ from 'lodash';
```

Comment:
If you did decide to keep lodash (although, I don't think you should), we should import them in a way that doesn't significantly impact the bundle size. I think doing this will result in >70kb and do something like this:

```suggestion
import keys from 'lodash/keys';
```

This is less important on the backend, but it's still an optimization that's worth it.